openy.profile

```diff
@@ -229,6 +250,58 @@ function openy_install_features(array &$install_state) {
   return ['operations' => $module_operations];
 }
 
+/**
+ * Create batch for installing search.
+ *
+ * @param array $install_state
+ *   Installation parameters.
+ *
+ * @return array
+ *   Batch.
+ */
+function openy_install_search(array &$install_state) {
+  \Drupal::state()->set('openy_hide_solr_config', '1');
+  $module = $install_state['openy']['search']['service'];
+  if (isset($install_state['openy']['search']['search_api_server'])) {
+    $server = $install_state['openy']['search']['search_api_server'];
+    if ($module == 'openy_search_api' && $server == 'solr') {
+      \Drupal::state()->set('openy_hide_solr_config', '0');
+    }
+  };
+
+  if (isset($install_state['openy']['search']['google_search_engine_id'])) {
+    \Drupal::state()->set('google_search_engine_id', $install_state['openy']['search']['google_search_engine_id']);
+  };
+  $files = \Drupal::service('extension.list.module')->getList();
+  if ($files[$module]->requires) {
+    $modules = array_merge(array_keys($files[$module]->requires), (array) $module);
+  }
+  foreach ($modules as $module) {
+    $module_operations[] = ['openy_enable_module', (array) $module];
+  }
+  return ['operations' => $module_operations];
+}
+
+/**
+ * Create batch for write google custom search id to configuration.
+ *
+ * @param array $install_state
+ *   Installation parameters.
+ *
+ * @return array
+ *   Batch.
+ */
+function openy_google_search(array &$install_state) {
```

Comment:
Can we save this config in `SearchSelectForm`, but not here?

Reply:
see comment above